server/cla.go

```diff
@@ -54,80 +59,108 @@ func (s *Server) checkCLA(pr *model.PullRequest) {
 		return
 	}
 
+	body, errCSV := s.getCSV()
+	if errCSV != nil {
+		return
+	}
+
+	if !isNameInCLAList(strings.Split(string(body), "\n"), username) {
+		comments, _, err := s.GithubClient.Issues.ListComments(context.TODO(), pr.RepoOwner, pr.RepoName, pr.Number, nil)
+		if err != nil {
+			mlog.Error("pr_error", mlog.Err(err))
```

Comment:
Nit pick: The error message should make clear what failed.

Reply:
Improved error message.